### ISSUE TITLE
Reduce Drop shadow opacity

### ISSUE THREAD

**Issue by msherman93** [DominicSherman/easy-budget-mobile#96]
Reduce drop shadow opacity by 30-40% and soften the shadow edges by about 20%

**Comment by DominicSherman** [comment]
Where? Everywhere?

**Comment by DominicSherman** [comment]
This is my current shadow object
```
export const shadow = {
    borderColor: Color.offBlack,
    borderWidth: 0,
    elevation: 3,
    shadowColor: Color.offBlack,
    shadowOffset: {
        height: 2,
        width: 0
    },
    shadowOpacity: 0.3,
    shadowRadius: 2
};export const shadow = {
    borderColor: Color.offBlack,
    borderWidth: 0,
    elevation: 3,
    shadowColor: Color.offBlack,
    shadowOffset: {
        height: 2,
        width: 0
    },
    shadowOpacity: 0.3,
    shadowRadius: 2
};
```

**Comment by DominicSherman** [comment]
Fixed with #101
